docs/sources/explore/_index.md

```diff
@@ -64,6 +64,44 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 To close the newly created query, click on the Close Split button.
 
+## Share Explore URLs
+
+When using Explore, the URL in the browser address bar is updated as you make changes to the queries. This allows you to share the URLs with others or bookmark them for later use.
+
+### Generating Explore URLs from external tools
+
+Because Explore URLs have a defined structure, you can build a URL from external tools and open it in Grafana. The URL structure is:
+
+```
+http://<grafana_url>/explore?panes=<panes>&schemaVersion=<schema_version>&orgId=<org_id>
+```
+
+where:
+
+- `org_id` is the organization ID
+- `schema_version` is the schema version (should be set to the latest version which is `1`)
+- `panes` is a url-encoded JSON object of panes, where each key is the pane ID and each value is an object matching the following schema:
+
+```
+{
+  datasource: string; // the pane's root datasource UID, or `-- Mixed --` for mixed datasources
+  queries: {
+    refId: string; // an alphanumeric identifier for this query, must be unique within the pane, i.e. "A", "B", "C", etc.
+    datasource: {
+      uid: string; // the query's datasource UID ie: "AD7864H6422"
+      type: string; // the query's datasource type-id, i.e: "loki"
+    }
+    // ... any other datasource-specific query parameters
+  }[]; // array of queries for this pane
+  range: {
+    from: string | number; // the start time, in milliseconds since epoch
+    to: string | number; // the end time, in milliseconds since epoch
+  }
+}
+```
+
+> **Note**: the `from` and `to` also accepts relative ranges defined in [Time units and relative ranges]({{< relref "../dashboards/use-dashboards/#time-units-and-relative-ranges" >}})
```

Comment:
I'm updating the note to use the admonition shortcode. We changed the Note format to be an admonition shortcode (see [Shortcodes](https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition-shortcode) content in the Writers' Toolkit). 
```suggestion
{{% admonition type="note" %}}
The `from` and `to` also accept relative ranges defined in [Time units and relative ranges]({{< relref "../dashboards/use-dashboards/#time-units-and-relative-ranges" >}}).
{{% /admonition %}}
```

Reply:
thanks!